apps/emqx_dashboard/src/emqx_dashboard_collection.erl

```diff
@@ -159,8 +162,11 @@ flush({Connection, Route, Subscription}, {Received0, Sent0, Dropped0}) ->
                diff(Sent, Sent0),
                diff(Dropped, Dropped0)},
     Ts = get_local_time(),
-    _ = mria:transaction(mria:local_content_shard(),
-                         fun mnesia:write/1, [#mqtt_collect{timestamp = Ts, collect = Collect}]),
+    Fun =
+        fun() ->
+            mnesia:dirty_write(?TAB_COLLECT, #mqtt_collect{timestamp = Ts, collect = Collect})
```

Comment:
why using dirty_write replace write in a transaction ?

Reply:
changed to
```erlang
mnesia:write(Tab, Record, Locked)
```